### DISCUSSION TITLE
Just Getting E-Mail from 'Updated By'

### DISCUSSION THREAD

**Discussion by silvanhornstein** [Budibase/budibase#8780] [Help] [answered]
Hey Dear community of my favourite LowCode Dev Tool!

I have a pretty stupid question and thought it might be easier asking than wasting another hour on trying to figure this out:

I am trying to access the E-Mail from a Table that is stored at 'Updated By'. However, I fail to do so, with Handlebars I am getting shown the full Uptated By Row, Javascript is not returning anything.

Happy about any advice, as I guess this is a nobrainer for most of you!
Best,
S.

**Comment by melohagan** [comment, marked as answer]
Hey @silvanhornstein 

You could create a formula column like this:

<img width="352" alt="Screenshot 2022-11-23 at 15 36 10" src="https://user-images.githubusercontent.com/101575380/203586944-88e7b1e1-a7d3-4a7a-b005-b95736988645.png">

You can see the binding is: `{{ Created By.0.email }}`. This is because the Created By field is a relationship type field, which is always an array type, even if it contains only one element.

<img width="191" alt="Screenshot 2022-11-23 at 15 37 33" src="https://user-images.githubusercontent.com/101575380/203587262-820bfba8-39fc-44bd-aa59-e07e7de68d88.png">

**Reply by silvanhornstein** [reply]
Hey @melohagan !
Thanks so much for the quick and well working fix. I wasnt aware of the formula function at all and now already have tons of usecases for me in my head. Best wishes and keep on rocking!

**Reply by melohagan** [reply]
Awesome! Here's the docs FYI: https://docs.budibase.com/docs/formula
